### DISCUSSION TITLE
End of page end of document

### DISCUSSION THREAD

**Discussion by DomenicoFormoso** [QuestPDF/QuestPDF#85] [General]
is it possible to place a text at the end of the page or at the end of the document?
Thanks

**Comment by MarcinZiabek** [comment]
Yes, it is possible in both cases 😁 This topic was already discussed [in another issue](https://github.com/QuestPDF/QuestPDF/discussions/47). Can you please specify your exact requirement?

**Comment by DomenicoFormoso** [comment]
I read the other issue and I fixed my problem. Thank you very much for offering your **precious** work.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
